src/lightning_app/core/constants.py

```diff
@@ -36,7 +36,7 @@
 LIGHTNING_COMPONENT_PUBLIC_REGISTRY = "https://lightning.ai/v1/components"
 LIGHTNING_APPS_PUBLIC_REGISTRY = "https://lightning.ai/v1/apps"
 ENABLE_STATE_WEBSOCKET = bool(int(os.getenv("ENABLE_STATE_WEBSOCKET", "0")))
-
+MULTI_WORKS_INTO_SINGLE_POD = bool(int(os.getenv("MULTI_WORKS_INTO_SINGLE_POD", "0")))
```

Comment:
Let's ensure that the same env_var is used everywhere to enable/disable this feature.

Reply:
Resolved.